src/bin/stratisd.rs

```diff
@@ -143,6 +148,8 @@ fn run() -> StratisResult<()> {
         // Refresh list of dbus fds to poll for every time. This can change as
         // D-Bus clients come and go.
         fds.truncate(engine_fds_end_idx);
+
```

Comment:
I guess that currently ```fds.truncate()``` becomes a no-op if dbus not enabled. But the comment just above it is now obviously out of place.